packages/api/src/routes/medical/internal-cq.ts

```diff
@@ -144,9 +161,180 @@ router.post(
   "/directory/organization",
   requestLogger,
   asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
     const body = req.body;
     const orgDetails = cqOrgDetailsOrgBizRequiredSchema.parse(body);
-    await createOrUpdateCQOrganization(orgDetails);
+    await createOrUpdateCQOrganization(cxId, orgDetails);
+
+    return res.sendStatus(httpStatus.OK);
+  })
+);
+
+/**
+ * GET /internal/carequality/directory/organization/v2/:oid
```

Comment:
Why v2 instead of updating the original ones

Reply:
Just thinking maybe we want the old ones there, but I removed them and removed the `v2` 👍